### PR TITLE
Fix issues which keep the temporary compat test fixutre

### PR DESCRIPTION
**Pull request by jakelandis**
This commit addresses a few issues which prevented the 
temporary compat test fixture from running. 

It is expected that the compat tests will fail with this PR.

### CODE REVIEW

**Comment by jakelandis** [comment]
@elasticmachine update branch

**Comment by jakelandis** [comment]
@pgomulka feel free to merge this PR to the feature branch.

File: qa/rest-compat-tests/build.gradle [hunk 1]
```diff
@@ -20,3 +20,5 @@ integTest.dependsOn(copyRestTestsResources)
 dependencies {
   compile project(':test:framework')
 }
+
+test.enabled = false
```
Comment (jakelandis) [inline qa/rest-compat-tests/build.gradle]: note - this line will likely conflict with [this line #53228](https://github.com/elastic/elasticsearch/pull/53228/files#diff-02871ed50004171fe02cef11d5b87e96R25)

Comment (pgomulka) [reply qa/rest-compat-tests/build.gradle]: I have only disabled it temporarily to see if this will try to continue the build

**Review by pgomulka — APPROVED** [review]
LGTM, we have 283 failures - which is expected

**Pull request merged**